packages/@angular/cli/utilities/dynamic-path-parser.ts

```diff
@@ -13,7 +13,9 @@ export interface DynamicPathOptions {
 export function dynamicPathParser(options: DynamicPathOptions) {
   const projectRoot = options.project.root;
   const sourceDir = options.appConfig.root;
-  const appRoot = path.join(sourceDir, 'app');
+
+  const p = options.appConfig.appRoot === undefined ? 'app' : options.appConfig.appRoot;
+  const appRoot = path.join(sourceDir, options.appConfig.appRoot);
```

Comment:
Creating local variable `p` with the determined root, but tot using it within the `join` on line 18

Reply:
Fixed it. Bad rebase.